src/wtc-perspective-card.js

```diff
@@ -922,13 +924,22 @@ class ClickablePerspectiveCard extends PerspectiveCard {
       // Set up the amount of rotation that needs to happen
       this.rotationAmount = Math.PI * -2;
 
-      // An empty endTween function for this tween
-      this.onEndTween = function() {};
+      this.onEndTween = function() {
+        // Transform style preserve 3d, and the translateZ were causing 
+        // the card image to pixelate on non retina monitors. Removing these
+        // whilst the card is open fixes that.
+        this.cardImage.style.transform = "initial";
```

Comment:
I wonder if `initial` has any implications here. Like if we wanted to transform the card in some other way, this inline style attribute would overwrite anything in the CSS. Is that okay?

If so, ignore! If not, maybe we should change this line to just remove the style attribute?

Reply:
I was thinking about this too and I'm really not sure there would ever be a use case for setting a custom transform style on the card image itself. Everything is set on the wrapper - this only has the `translateZ` attached so that you can see the backside of the card during the opening animation.

Happy to remove the transform style though if you think it may be an issue?

Reply:
I think this is probably fine, but it might be more portable if we were to use a className for this purpose. Any issues with that?

Reply:
Cool, we can ignore my note then. `className` could be useful, in case anything ever had to be overwritten

Reply:
Updated to have a wrapping `perspective-card--is-open` class when opened and targeting the style changes in the CSS. Have tested locally. Let me know if you want me to demo @liamegan & @andrewrubin!